### PR TITLE
Feature/csd 173/return status result

### PR DESCRIPTION
**Pull request by QasimAbbas**

### CODE REVIEW

File: Snowplow/SPDefaultNetworkConnection.m [hunk 1]
```diff
@@ -172,7 +172,10 @@ - (NSURL *)url {
             
             dispatch_semaphore_wait(sem, DISPATCH_TIME_FOREVER);
 
-            NSLog(@"ljw statusCode=%ld", [httpResponse statusCode]);
+            NSLog(@"ljw statusCode=%ld", (long)[httpResponse statusCode]);
+            [[self->_callback responseStatus] setValue:[NSString stringWithFormat:@"%ld", (long)[httpResponse statusCode]] forKey:@"statusCode"];
+            [[self->_callback responseStatus] setValue:urlRequest.URL.absoluteString forKey:@"url"];
```
Comment (LamarDataSherpas) [inline Snowplow/SPDefaultNetworkConnection.m:177]: This was suppose to be a method not a property. It's probably ok, as long as we can use didSet on the property to trigger an asynchronous client call back. Does didSet work when calling a swift property from Objective C?

Comment (QasimAbbas) [reply Snowplow/SPDefaultNetworkConnection.m]: Adjustments to make this a method have been made.

File: SnowplowTracker.podspec [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "SnowplowTracker"
-  s.version          = "1.6.6"
+  s.version          = "1.6.7dev-qa1"
   s.summary          = "Snowplow event tracker for iOS, macOS, tvOS, watchOS for apps and games."
```
Comment (LamarDataSherpas) [inline SnowplowTracker.podspec]: I'm not sure what this s.version is for, just remember our versioning scheme.

**Review by LamarDataSherpas — APPROVED** [review]

**Comment by LamarDataSherpas** [comment]
fix the version and merge

**Comment by LamarDataSherpas** [comment]
Remember the last version was 1.6.0-1.0.0
Your version will be 1.6.0-1.0.1

**Pull request merged**